src/covertool.erl

```diff
@@ -144,7 +144,13 @@ package_name(AppName, PrefixLen, Module)
                             "." ->
                                 [];
                             DirName ->
-                                string:tokens(DirName, "/")
+                                case string:tokens(DirName, "/") of
+                                    ["_build", _Profile, "lib", AppNameStr, "src" | Rest] ->
```

Comment:
@tsloughter @idubrov sorry to bug you again twice in an evening, but curious if you agree with this approach here. This is a very narrowly structured pattern and might be fragile with respect to some rebar3 magic I am not aware of. Ever since #35 was merged, the package naming has been filled with this stuff. I think it may be because of [this](https://github.com/idubrov/covertool/pull/35/files#diff-86b6e7e2c954ae9c28a65b1420b58933L273) change, which I think maybe needs to just be reverted, or perhaps fixed in another way.

Reply:
I don't know that I am prepared to be the sole maintainer, but it would be cool to get it moved to an organization (like https://github.com/covertool/covertool) and have a few maintainers added. I would be happy to be part of that.

Reply:
@nalundgaard I'm happy to help maintain.

Reply:
@idubrov can you add @tsloughter and @tverlaan as covertool members/owners, and/or make me an "owner" so that I can manage organization membership?

I'll make a PR later today to address this package name issue and hopefully get the reports back to a good state for both rebar3 & mix.